File: challenges/1-media-alunos/c++/lucashe4rt/media.cpp
#include <iostream>

int main()
{
    float primeiraNota, segundaNota;

    std::cout << "Insira a primeira nota: ";
    std::cin >> primeiraNota;
    std::cout << "Insira a segunda nota: ";
    std::cin >> segundaNota;

    std::cout << "A média do aluno é: " << (primeiraNota + segundaNota) / 2 << std::endl;
}

File: challenges/1-media-alunos/typescript/riicksouzaa/src/media.ts
import * as readline from 'readline'

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Qual a primeira nota do aluno? ', (answer1) => {
    rl.question('Qual a segunda nota do aluno? ', (answer2) => {
        var result = ((+answer1) + (+answer2)) / 2;
        console.log(`A média das notas é ${result}`);
        rl.close();
    });
});